File: problem11/Solution.java
public class Solution {
	public static void main(String[] args) {
		System.out.println(isSubsequence("axc", "ahbgdc"));
	}
	public static boolean isSubsequence(String s, String t) {
        int i, j, slen = s.length(), tlen = t.length();
		for (i = 0, j = 0; i < slen && j < tlen; j++) {
        	if (s.charAt(i) == t.charAt(j))
				++i;
        }
		if (i == slen)
			return true;
		return false;
    }
}

File: problem11/Solution.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        i, j = 0, 0
        slen, tlen = len(s), len(t)
        while i < slen and j < tlen:
            if s[i] == t[j]:
                i += 1
            j += 1
        return i == slen

